webapp/src/components/modals/channel_subscriptions/edit_channel_subscription.tsx

```diff
@@ -329,6 +416,15 @@ export default class EditChannelSubscription extends PureComponent<Props, State>
         }
     };
 
+    handleTemplateChange =(_: any, templateID: string) => {
```

Comment:
```suggestion
    handleTemplateChange =(_: any, templateId: string) => {
```
Also, can we give a proper type?